app/assets/stylesheets/agents.scss

```diff
@@ -82,19 +82,28 @@ $cards-gutter: 0.5em;
 }
 
 .agent-status-color {
-  background-color: #37c837;
   padding: 3%;
   width: 1em;
   height: 1em;
   border-radius: 4px;
 }
 
-.agent-name {
-  display: inline-block;
+.agent-status-color-default {
+  background-color: black;
 }
 
-.agent-status {
+.agent-status-color-green {
+  background-color: #37c837;
 }
 
-.status-timer {
+.agent-status-color-yellow {
+  background-color: yellow;
```

Comment:
More pastel colors fits better with rest of the colors. Maybe use colors from #9?